Bugzilla/CGI.pm

```diff
@@ -561,6 +561,10 @@ sub header {
     # the MIME type away from the declared Content-Type.
     $headers{'-x_content_type_options'} = 'nosniff';
 
+    # Add Referrer-Policy (sic) header to prevent browsers sending
+    # Referer (sic) headers to external websites.
+    $headers{'-referrer_policy'} = 'same-origin';
```

Comment:
This would cause more issues than it solves. It would set the document policty to any one of the three values *at random* (as per [the standard algorithm](https://www.w3.org/TR/referrer-policy/#parse-referrer-policy-from-header) that doesn’t specify a sorting order). 

I did a quick check, and not a single website in the HTTP Archive has the `Referrer-Policy` value of `same-origin, no-referrer, none`. I ran out of credits there just now so I can’t check if any value at all contains a comma.

Reply:
The algorithm for Referrer Policy does specify an order:
https://www.w3.org/TR/referrer-policy/#unknown-policy-values

So `Referrer-Policy: no-referrer, strict-origin-when-cross-origin` is perfectly valid, as is the policy above.